policy/modules/contrib/rsync.te

```diff
@@ -35,6 +35,12 @@ gen_tunable(rsync_anon_write, false)
 ## </desc>
 gen_tunable(rsync_full_access, false)
 
+## <desc>
+##	<p>
+##	Allow rsync sys_admin capability
```

Comment:
Can you please add the information here that this is needed for setting the `trusted.*` extended attributes? I think it would be helpful to have it there, not just in the git history.

Reply:
Description enhanced.